Tasks/zip.js

```diff
@@ -1,18 +1,13 @@
 // Zip two arrays, [1, 2] and [3, 4] -> [[1, 3], [2, 4]]
+'use strict';
 
-const zip = function (a = [], b = []) {
-  let i = 0;
-  j = 0;
-  for (x of b) {
-    CELL = [a[i++], x];
-    if (i < j) {
-      delete a[i++];
-    } else {
-      (() => (b[j++] = CELL))();
-    }
-    if (CELL[0] == undefined) b.length -= 1
+const zip = (arr1, arr2) => {
+  const result = [];
+  for (let i = 0; i < arr1.length && i < arr2.length; i++) {
```

Comment:
Alternative solution is to iterate over iterable returned from `entries()`
```suggestion
  for (const [i, value] of arr1.entries()) {
```

Reply:
added alternative solution